Makefile

```diff
@@ -377,7 +377,7 @@ create-workload-cluster: $(ENVSUBST) $(KUBECTL) ## Create a workload cluster.
 	timeout --foreground 1800 bash -c "while ! $(KUBECTL) get secrets -n default | grep $(CLUSTER_NAME)-kubeconfig; do sleep 1; done"
 	# Get kubeconfig and store it locally.
 	$(KUBECTL) get secret/$(CLUSTER_NAME)-kubeconfig -n default -o json | jq -r .data.value | base64 --decode > ./kubeconfig
-	$(KUBECTL) -n default wait --for=condition=Ready --timeout=10m cluster "$(CLUSTER_NAME)"
+	$(KUBECTL) -n default wait --for=condition=Ready --timeout=30m cluster "$(CLUSTER_NAME)"
```

Comment:
Is it worth parameterizing this, like `--timeout=$(CLUSTER_TIMEOUT)`? This is a popular Makefile target and this may change the overall duration of e2e test failures, or surprise users who were accustomed to failure after 10 minutes.

Reply:
I asked @jackfrancis about this and we decided to just leave this for now and follow up later b/c the timeouts seem to be hard-coded all over the place like on line 377 where it's 1800 seconds (30 min). The idea is to get this working for now, and then try to standardize later.